platzky/models.py

```
@@ -4,6 +4,22 @@
 from pydantic import BaseModel
 
 
+class CmsModule(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)